gputop-csv/gputop-csv.js

```diff
@@ -134,6 +136,59 @@ GputopCSV.prototype.list_metric_set_counters = function(metric) {
     stderr_log.log("\nALL: " + all);
 }
 
+var ctx_hw_id_ = [];
+var vgpu_id_ = [];
+var map_vgpuID_hwID = [0];
+
+GputopCSV.prototype.get_vgpu_id = function() {
+    var vgpu_id;
+    for (var i = 0; i < vgpu_id_.length; i++) {
+        if (vgpu_id_[i] === parseInt(args.vgpu)) {
+            vgpu_id = vgpu_id_[i];
+                 break;
```

Comment:
Please align this break statement.

Reply:
Sorry for my mistake, I have corrected it.